Package.swift

```diff
@@ -65,7 +65,6 @@ let package = Package(
           "LSPTestSupport",
           "SourceKitLSP",
           .product(name: "ISDBTestSupport", package: "IndexStoreDB"),
-          .product(name: "tibs", package: "IndexStoreDB"), // Never imported, needed at runtime
```

Comment:
`tibs` is already listed as a dependency of `ISDBTestSupport`, but it causes linking issues in test targets. As an executable it can't be linked to a test executable due to `multiple definition of 'main'` error.